src/CodeGen_GPU_Host.cpp

```diff
@@ -191,8 +191,8 @@ void CodeGen_GPU_Host<CodeGen_CPU>::compile_func(const LoweredFunc &f,
         std::vector<char> kernel_src = gpu_codegen->compile_to_src();
 
         Value *kernel_src_ptr =
-            CodeGen_CPU::create_constant_binary_blob(kernel_src,
-                                                     "halide_" + function_name + "_" + api_unique_name + "_kernel_src");
+            CodeGen_CPU::create_binary_blob(kernel_src,
```

Comment:
The only call site that doesn't use the default uses a named variable 'constant'...

Also, I don't see what is wrong with the name. It's certainly not that different from the old name. Open to suggestions here, just not sure how it should be changed.